src/sonic-host-services/scripts/caclmgrd

```diff
@@ -201,6 +206,12 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
         tcp_flags_str = tcp_flags_str[:-1]
         return tcp_flags_str
 
+    def update_feature_present(self):
+        feature_tb_info = self.config_db_map[DEFAULT_NAMESPACE].get_table(self.FEATURE_TABLE)
+        if feature_tb_info:
+            for k, v in feature_tb_info.items():
+                self.feature_present[k] = True
```

Comment:
>feature_present [](http://example.com/codeflow?start=21&length=15)

Seems this is a generic oneshot cache. Suggest leverage common library like `cached_property`. ref: [https://www.geeksforgeeks.org/python-functools-cached_property/](https://www.geeksforgeeks.org/python-functools-cached_property/)

Reply:
thank you for the pointer. Currently, we are updating this feature_present map with the features that are present only once during initialization so it is not repeatedlyinvoked.
Also, if we use cached_property, we will invoke the function once, which could be in per-thread context, which might cause issues. So not using cached_property in this case.